lib/netif.ml

```diff
@@ -18,7 +18,8 @@
 open Result
 open V1.Network
 
-let log fmt = Format.printf ("Netif: " ^^ fmt ^^ "\n%!")
+let src = Logs.Src.create "netif" ~doc:"Mirage Solo5 network module"
+module Log = (val Logs.src_log src : Logs.LOG)
```

Comment:
yes, since solo5 supports only a single netif atm, this is a non-issue, I wanted to raise awareness nevertheless.

please also update the opam in the mirage-dev repository (to add the logs dep)

it's your bug ;) -- feel free to PR to mirage (or wherever `tapX` originates from)